.pre-commit-config.yaml

```diff
@@ -50,6 +50,7 @@ repos:
       args:
         - "--config=setup.cfg"
         - '--extend-ignore=D1'
+        - '--max-string-usages=10'
```

Comment:
This should be removed. Any configuration settings for flake8 should be managed via setup.cfg